shared/chat/conversation/info-panel/min-writer-role/container.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as Container from '../../../../util/container'
+import * as Constants from '../../../../constants/chat2'
+import * as Chat2Gen from '../../../../actions/chat2-gen'
+import * as Types from '../../../../constants/types/chat2'
+import * as TeamTypes from '../../../../constants/types/teams'
+import * as TeamConstants from '../../../../constants/teams'
+import MinWriterRole from '.'
+
+type OwnProps = {
+  conversationIDKey: Types.ConversationIDKey,
+  isSmallTeam: boolean,
+}
+
+const emptyProps = {
+  canSetMinWriterRole: false,
+  minWriterRole: 'reader',
+}
+
+const mapStateToProps = (state: Container.TypedState, ownProps: OwnProps) => {
+  const meta = Constants.getMeta(state, ownProps.conversationIDKey)
+  if (!meta.teamname) {
+    return emptyProps
```

Comment:
What is `emptyProps`?

Reply:
I don't think this will ever happen since `meta.teamname` is used in the info panel to decide whether this will render. But if this does happen it'll render the non-admin view from the original comment.

Reply:
Oh, I just didn't see where it was defined was all.